paymentmethod.go

```diff
@@ -39,6 +39,23 @@ const (
 	PaymentMethodCardBrandVisa       PaymentMethodCardBrand = "visa"
 )
 
+// PaymentMethodCardNetwork is the list of allowed values to represent the network
+// used for a card-like transaction.
+type PaymentMethodCardNetwork string
```

Comment:
I made a call here on where to set the enum. I am not sure this is the right decision in the library. In a "normal" world we'd name if `ChargePaymentMethodDetailsCardNetwork` and it'd be specific, and duplicated for CardPresent, but in the future the network will be chosen for a PI and that will make us rename it.

Long story short: we need to figure out clear rules on how to name enums.